### DISCUSSION TITLE
ability to set time on camera?

### DISCUSSION THREAD

**Discussion by doctorevil30564** [openmiko/openmiko#107] [Q&A] [answered]
Forgive me if this has been asked anywhere else , I looked but didn't see this question. I used my google fu skills as well and didn't find anything for setting the correct time and time zone on openmiko

Would it be possible to add the ability to sync with a specified NTP server and show the correct time on my v2 cameras?

I currently have the time stamp turned off, but I would like to be able to have a time stamp running in the image when I am watching it on my octoprint server

**Comment by martinbutt** [comment, marked as answer]
`ntpd` is preinstalled and runs on startup. The NTP server defaults to `time.google.com`.

If it is showing the incorrect timezone, then you can follow [these instructions](https://github.com/openmiko/openmiko/wiki/Setting-the-Time-Zone).

**Reply by doctorevil30564** [reply]
excellent, thank you :)
